docs/api.md

```
@@ -0,0 +1,89 @@
+**Response:**
+```json
+{
+  "token": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9..."
+}
+```
+
+---
+
+### OAuth Login (Google / GitHub)
+
+G-Secure supports OAuth via NextAuth.js. Redirect the user to:
```

markdownlint-cli2 (0.22.1)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)